### PR TITLE
add script for wsl startup

### PR DESCRIPTION
**Pull request by bhowl**
Bash script using i/o redirection, grep, and sed to automate the input of wsl's 'inet IP address' into client.mjs's 'window.location.hostname'

### CODE REVIEW

**Comment by howl0893** [comment]
@bradleyhh 
man sed is a nasty sunuvabitch

Shouldn't the wsl.sh script be in the play folder **OR** line 3 be: `sed -i "s,window.location.hostname,\"${ip}\"," ./play/public/js/client.mjs` though?

Also what about adding an output just so the user knows the script did something... maybe add @ line 2:
`echo "changing hostname to: $(<./tmp-ip.txt)"`

File: wsl.sh [hunk 1]
```diff
@@ -0,0 +1,4 @@
+ifconfig | grep inet | sed -e '2,4d' -e 's/  netmask/\n\t  netmask/' -e 's/broadcast/\n\tbroadcast/' | sed -e '2,3d' -e 's/        inet //' > ./tmp-ip.txt
+ip=$(cat ./tmp-ip.txt)
+sed -i "s,window.location.hostname,\"${ip}\"," ./public/js/client.mjs
```
Comment (howl0893) [inline wsl.sh]: Should be ./play/public/js/client.mjs if in the root folder, no?

**Comment by bhowl** [comment]
> @bradleyhh 
> 
> man sed is a nasty sunuvabitch
> 
> 
> 
> Shouldn't the wsl.sh script be in the play folder **OR** line 3 be: `sed -i "s,window.location.hostname,\"${ip}\"," ./play/public/js/client.mjs` though?
> 
> 
> 
> Also what about adding an output just so the user knows the script did something... maybe add @ line 2:
> 
> `echo "changing hostname to: $(<./tmp-ip.txt)"` 

Yes it should be in play folder! My bad, it was in play folder in my local repo. I still have to set up remote repos in terminal (I think) so I just uploaded this file quickly on the web. Or line 3 edit could work too, I think it should go in the same spot as the ./startup script. 

Good idea on adding output. We could also add to line 5 `head ./public/js/client.mjs` to show the top of the file. This will show that the change was made to client.mjs

And yes sed is nasty looking but can be mighty powerful

**Comment by bhowl** [comment]
I changed the permissions on this repo so that I could 'write', which allows any of us to clone, pull, and push to this repo from the command line.

**Review by bhowl — COMMENTED** [review]
Moved this script to play/ and added echo lines. Looks good to merge to me

**Review by howl0893 — APPROVED** [review]

**Pull request merged**